Factorial/Factorial.c

```diff
@@ -0,0 +1,35 @@
+/* This code is used to find factorial 
+of large numbers*/
+#include<stdio.h>
+int main()
+{
+    int n,a[200000],i,j,k,l,m;
+    printf("ENTER THE NUMBER\n");
+	scanf("%d",&n);
+        m=1;
+        a[0]=1;
+        for(j=2;j<=n;j++)
```

Comment:
correct the indentation (keep four spaces) and keep space before and after operators